libs/ledger-live-common/src/account/serialization.ts

```diff
@@ -797,15 +797,23 @@ export function fromAccountRaw(rawAccount: AccountRaw): Account {
         (rawAccount as BitcoinAccountRaw).bitcoinResources
       );
       break;
-    case "algorand":
-      (res as AlgorandAccount).algorandResources = fromAlgorandResourcesRaw(
-        (rawAccount as AlgorandAccountRaw).algorandResources
-      );
+    case "algorand": {
+      const algoResourcesRaw = (rawAccount as AlgorandAccountRaw)
+        .algorandResources;
+      if (algoResourcesRaw)
+        (res as AlgorandAccount).algorandResources =
+          fromAlgorandResourcesRaw(algoResourcesRaw);
       break;
+    }
```

Comment:
Just curious, I see you changed this pattern for several coins, what is this fixing?

Reply:
mock don't have any algorandResources (and other family don't have `familyResources`) unless we are putting fake data on every coins. But this couldn't work because of typing, DataTest raw is bound to AccountRaw and we can't put any typing unless we are do a long rework on every bridge.integration. But it's out of this ticket scope + after talking with Hedi, it's seems that it can be not worth the time thanks to coin modularization